pyhf/readxml.py

```diff
@@ -59,6 +59,9 @@ def process_sample(
     data, err = import_root_histogram(rootdir, inputfile, histopath, histoname)
 
     modifiers = []
+    # first check if we need to add lumi modifier for this sample
+    if sample.attrib.get("NormalizeByTheory", "False") == 'True':
+        modifiers.append({'name': 'lumi', 'type': 'lumi', 'data': None})
```

Comment:
we need to somehow pass the lumi uncertainty in.. this is a bit tricky since it's not defined by the tag in the sample, but rather the toplevel measurement object.. so maybe we store that value when we read the toplevel object and then inject it into `data` here... or we set the lumi uncertainty from somewhere

Reply:
Yep, that's the part I'm slightly stuck on.